geometry_msgs/Pose pose
int32 instance_id
uint32 class_id
